primer/index.bs

```diff
@@ -235,6 +235,8 @@ Response Body:
 }
 ```
 
+Issue: OIDC Federation defines `client_registration_types_supported`, should it be addedhere, to `.well-known/oidc-federation` or both?
```

Comment:
This should definitely be added to OpenID Metadata resource (`/.well-known/openid-configuration`) for OPs that support clients from a federation.

This MAY be added to OpenID Provider metadata that also participate in a federation (note the resource path: `.well-known/openid-federation`).

As I have been considering this change, I have been imagining the relying parties participating in an OIDC federation, but that it would entirely optional for an OpenID Provider to participate in a federation. Or are you thinking that this would be a requirement for Solid OIDC Providers as well?

Reply:
Thanks! I'm going to add it to `/.well-known/openid-configuration`.

I think we should start with the minimum required, probably enough for RPs to act as entities in federation.